src/hooks/usePodOwnerProfile/index.js

```diff
@@ -25,14 +25,17 @@ import { joinPath } from "../../stringHelpers";
 import useAuthenticatedProfile from "../useAuthenticatedProfile";
 import usePodRoot from "../usePodRoot";
 import useFetchProfile from "../useFetchProfile";
+import usePodOwner from "../usePodOwner";
 
 export default function usePodOwnerProfile() {
   const [profile, setProfile] = useState();
   const [error, setError] = useState();
   const router = useRouter();
   const decodedResourceUri = decodeURIComponent(router.query.iri);
+  const { podOwnerWebId } = usePodOwner(router.query.iri);
```

Comment:
This will return null while the request is being made, causing the first load to attempt to load profile/card#me. You may need to return a `requestInProgress` boolean so that, later on in the effect to setProfile, you only set the profile iri if the request is _not_ in progress.

Reply:
You should also extend the tests for `userPodOwnerProfile` with a scenario that successfully uses `usePodOwner`.